README.md

```diff
@@ -34,92 +34,111 @@ sudo echo "127.0.0.1 pulp.content" | sudo tee -a /etc/hosts
 ### Start dependency containers
 
 ```sh
-$ make compose-up
+make compose-up
 ```
 
 ### Import Public Repos
 
 ```sh
-$ make repos-import
+make repos-import
+```
+
+### For local development, if you want less Redhat repos try"
+
+```sh
+OPTIONS_REPOSITORY_IMPORT_FILTER=small make repos-import
 ```
 
 ### Run the server!
 
 ```sh
-$ make run
+make run
 ```
 
 ###
 
 Hit the API:
 
 ```sh
-  $ curl -H "$( ./scripts/header.sh 9999 1111 )" http://localhost:8000/api/content-sources/v1.0/repositories/
+curl -H "$( ./scripts/header.sh 9999 1111 )" http://localhost:8000/api/content-sources/v1.0/repositories/
 ```
 
 ### Stop dependency containers
 
 When its time to shut down the running containers:
 
 ```sh
-$ make compose-down
+make compose-down
 ```
 
 And clean the volume that it uses by (this stops the container before doing it if it were running):
 
 ```sh
-$ make compose-clean
+make compose-clean
 ```
 
 > There are other make rules that could be helpful, run `make help` to list them. Some are highlighted below
 
+## Playwright testing
+
+- Ensure that the backend server is running
+- Make sure you are currently on the correct [node version](_playwright-tests/.nvmrc), you can do this easily with nvm.
+  `cd _playwright-tests` and `nvm use`
+- You also need to copy the [env](_playwright-tests/example.env) file and create your own file at: \_playwright-tests/.env
```

Comment:
```suggestion
- Copy the [env](_playwright-tests/example.env) file and create your own file at: \_playwright-tests/.env
```

Reply:
Would be nice to say more about that file's contents, perhaps at least "For local testing, only BASE_URL is required"

Reply:
yeah...